src/chart/draw/consolidateData/setDefaults.js

```diff
@@ -15,6 +15,9 @@ export default function setDefaults() {
     this.config.x.type = this.config.x.type || 'linear';
     this.config.y.type = this.config.y.type || 'linear';
 
+    this.config.x.range_band = this.config.x.range_band || this.config.range_band;
```

Comment:
Do we need to set a numeric default for range_band first? Or is that handled elsewhere? 

`this.config.range_band = this.config.range_band || 20`
...

Reply:
When `range_band` isn't specified the marks are sized to fit the canvas.  Otherwise `range_band` dictates the size of the marks and also the size of the canvas.